# `python:3.8.0a2-alpine3.9`

## Docker Metadata

- Image ID: `sha256:61e9008a7ea71568998a68db42e16ae61a6eafe304a35cc3db32a9bdad22f0af`
- Created: `2019-02-28T01:42:04.503414596Z`
- Virtual Size: ~ 93.52 Mb  
  (total size of all layers on-disk)
- Arch: `linux`/`amd64`
- Command: `["python3"]`
- Environment:
  - `PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`
  - `LANG=C.UTF-8`
  - `GPG_KEY=E3FF2839C048B25C084DEBE9B26995E310250568`
  - `PYTHON_VERSION=3.8.0a2`
  - `PYTHON_PIP_VERSION=19.0.3`
